client/orderbook/epochqueue.go

```diff
@@ -27,10 +28,11 @@ type epochOrder struct {
 type EpochQueue struct {
 	orders map[order.OrderID]*epochOrder
 	mtx    sync.RWMutex
+	log    dex.Logger
 }
 
 // NewEpochQueue creates a client epoch queue.
-func NewEpochQueue() *EpochQueue {
+func NewEpochQueue(logger dex.Logger) *EpochQueue {
```

Comment:
`log` field not set by constructor

Reply:
Seems there's only one `eq.log` in this type's methods, and it's not hit by test coverage (hence to panic in tests).  Wondering if this type needs a logger at all...

Reply:
I feel similarly, but it's not very heavy and it's nice to have it for debugging.